components/header/Header.tsx

```diff
@@ -109,22 +113,68 @@ const Sticky = styled(LayoutHeader)`
         height: initial; /* Ant sets a height, we need to unset that. */
         padding: 0;
         transition: top 200ms ease-in-out;
+        line-height: initial;
     }
 `;
 
-export const HeaderColumns = styled(Columns)`
-    width: 100%;
-    align-items: center;
-`;
-
-export const HeaderTitle = styled.h5`
+export const Label = styled.h5`
```

Comment:
```suggestion
export const AppName = styled.h5`
```

I think `Label` is overloaded. There's an HTML `<label />` element. I think `AppName` makes it clear, we could also do `Name`, but that's a little less clear I think.